### PR TITLE
Feat: implement unit test getAll()

### PR DESCRIPTION
**Pull request by kellyCSalazar**
## Describe your changes
Create unit test getAll()

## Issue ticket number and link
https://trello.com/c/ovU3G2sE

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: services/events.service.js [hunk 1]
```diff
@@ -357,7 +363,14 @@ class EventsService extends BaseService {
         queryEvents['headquarter'] = new ObjectId(headquarterId)
       }
 
-      return await this.collection.find(queryEvents).toArray()
+      const data = await this.collection.find(queryEvents)
+
+      for await (const doc of data) {
+        events.push(doc)
+      }
```
Comment (jlora-mck) [inline services/events.service.js:370]: Is data a promise ? if so can we explore not doing this and instead do a `promise.all` ?

File: tests/fixtures/event.mock.js [hunk 2]
```diff
@@ -15,28 +14,29 @@ class MockEvent {
...
+    return event
   }
 
   generateList(eventsData) {
-    const events = [];
+    const events = []
+
     for (let i = 0; i < eventsData.length; i++) {
-      events.push(this.generate(eventsData[i]));
+      events.push(this.generate(eventsData[i]))
```
Comment (jlora-mck) [inline tests/fixtures/event.mock.js:35]: can we use for .. of

File: tests/services/events.service.spec.js [hunk 3]
```diff
@@ -67,7 +55,7 @@ test.afterEach(() => {
   sandbox && sandbox.restore()
 })
 
-test.serial('Create event', async t => {
+test.skip('Create event', async t => {
```
Comment (jlora-mck) [inline tests/services/events.service.spec.js:58]: is this ok ? skipping this test ?

File: tests/services/events.service.spec.js [hunk 4]
```diff
@@ -143,7 +126,7 @@ test.skip('Do list all events with year and headquarter', async t => {
   })
 })
 
-test.serial('Get event', async t => {
+test.skip('Get event', async t => {
```
Comment (jlora-mck) [inline tests/services/events.service.spec.js:129]: why we are skipping test again ? if we need to skip test please add a comment to the PR or add an inline comment so we know

**Review by jlora-mck — CHANGES_REQUESTED** [review]
Left a couple comment, please address all fors in the PR if possible :D but in general great work!

File: services/events.service.js (hunk 1 above)
Comment (jose-sal-y-rosas) [reply services/events.service.js:370]: @kellyCSalazar I think you can use toArray here instead of that for.

**Review by jose-sal-y-rosas — COMMENTED** [review]
Leave a comment

File: services/events.service.js (hunk 1 above)
Comment (kellyCSalazar) [reply services/events.service.js]: Changed for toArray

**Review by anversa-pro — APPROVED** [review]
Great work Kelly!

File: tests/fixtures/event.mock.js [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
+const MockHeadquarter = require('./headquarter')
 
 class MockEvent {
-  constructor() {
-  }
+  constructor() {}
 
-  static generate(eventData) {
+  generate(eventData) {
     let event = {
```
[the remark this answers is not in the document]
Comment (kellyCSalazar) [reply tests/fixtures/event.mock.js]: Changed for const

File: tests/fixtures/event.mock.js (hunk 2 above)
Comment (kellyCSalazar) [reply tests/fixtures/event.mock.js:35]: Changed for .map()

File: tests/services/events.service.spec.js (hunk 4 above)
Comment (kellyCSalazar) [reply tests/services/events.service.spec.js:132]: The tests will be passed for the moment, they will be done as the respective endpoint is done

File: tests/services/events.service.spec.js [hunk 6]
```diff
@@ -96,27 +86,23 @@ test.serial('Create event', async t => {
...
 })
 
-// TODO: Review mock data to run this test
-test.todo('Do list all events without params', async t => {
-  //const error = await t.throwsAsync(() => eventsService.doList({}), { instanceOf: Error })
-
+test.serial('Do list all events without params', async t => {
   let eventsData = await eventsService.doList({})
 
+  console.log('EVENTSDATA', eventsData)
```
Comment (FelipeLosada1996) [inline tests/services/events.service.spec.js:92]: remove console.log

File: tests/services/events.service.spec.js (hunk 3 above)
Comment (kellyCSalazar) [reply tests/services/events.service.spec.js:60]: The tests will be passed for the moment, they will be done as the respective endpoint is done

File: tests/services/events.service.spec.js (hunk 6 above)
Comment (kellyCSalazar) [reply tests/services/events.service.spec.js]: Delete consolo.log

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jlora-mck — APPROVED** [review]

**Pull request merged**